pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -170,6 +172,10 @@ func (proxy *DataSourceProxy) getDirector() func(req *http.Request) {
 			req.Header.Add("Authorization", dsAuth)
 		}
 
+		if proxy.cfg.SendUserHeader && !proxy.ctx.SignedInUser.IsAnonymous {
+			req.Header.Add("X-Grafana-User", proxy.ctx.SignedInUser.Login)
```

Comment:
I assume `proxy.ctx.SignedInUser.Login` is correct value but wasn't exactly sure.

Reply:
yea `Login` is correct :+1: